Added a `angular_radius` convenience property to `Helioprojective` (#5191)
sunpy/sunpy: changelog/5191.trivial.rst

> Added angular radius as a property to `HelioProjective`.

Reviewer: ```suggestion
The `~sunpy.coordinates.frames.Helioprojective` frame now has the convenience property ``angular_radius`` to return the angular radius of the Sun as seen by the observer.
```

> Added a Convenience property of angular radius in the HelioProjective class in Frames.py

Reviewer 2: ```suggestion
Added angular radius as a property to `HelioProjective`.
```
